src/app/static/src/app/components/plots/bubble/BubblePlot.vue

```diff
@@ -452,9 +452,9 @@
                 this.$emit("update-colour-scales", newColourScales);
             },
             updateSizeScale: function (scale: ScaleSettings) {
-                const newColourScales = {...this.sizeScales};
-                newColourScales[this.selections.sizeIndicatorId] = scale;
-                this.$emit("update-size-scales", newColourScales);
+                const newSizeScales = {...this.sizeScales};
+                newSizeScales[this.selections.sizeIndicatorId] = scale;
+                this.$emit("update-size-scales", newSizeScales);
```

Comment:
Looks good, render well together. 

I found one weird behaviour in the console though, I believe it has something to do with the scale size. When a user selects an option from size indicator drop down menu which has empty circle radius, the bubble returns NaN. I think we could have a conditional statement checking if selections have radius before trying to display or compute bubbles in min/max control legend.

Reply:
Ah, great spot. Hm, interesting, they should really all have a calculated radius for all values. Can you tell me how to recreate this with the test data, or maybe upload a screenshot so I can see all the settings?

Reply:
<img width="1319" alt="Screenshot 2020-11-26 at 18 24 06" src="https://user-images.githubusercontent.com/4240748/100383100-b9d2a100-3014-11eb-86bf-0c0b33105131.png">

Reply:
This bug isn't actually caused by your ticket. Its from master.

Reply:
Yeah, it's because when there's no data, the data range includes NaN and it attempts to draw SizeLegend bubbles with radius NaN... I've fixed it in this branch, though actually your branch to hide the legends when no data should make this go away too I think.